.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,20 @@
+name: Build image
+on:
+  pull_request:
+    branches:
+      - develop
+      - 'release/**'
+jobs:
+  plugin-build-test:
```

Comment:
Should we use the same filter here such as on the rust build?

Reply:
Added them here